scripts/src/javatests/org/oppia/android/scripts/coverage/RunCoverageTest.kt

```diff
@@ -10,18 +10,13 @@ import org.oppia.android.scripts.common.CommandExecutorImpl
 import org.oppia.android.scripts.common.ScriptBackgroundCoroutineDispatcher
 import org.oppia.android.scripts.testing.TestBazelWorkspace
 import org.oppia.android.testing.assertThrows
-import java.io.ByteArrayOutputStream
 import java.io.File
-import java.io.PrintStream
 import java.util.concurrent.TimeUnit
 
 /** Tests for [RunCoverage]. */
 class RunCoverageTest {
```

Comment:
### Todo:
to add test for
saving proto
exemptions check
multiple hits on a same line